### ISSUE TITLE
Baxxid unable to take pills, water.

### ISSUE THREAD

**Issue by TheKnightofAura** [ScavStation/Baystation12#224]

**Comment by ZirconiumWit** [comment]
Dear people services

Today I got dangerously close to a baxxid's mouth.  I fear the day that I will have to perform an oral surgery on such a gargantuan creature.  Anyway.  We determined a problem of the utmost importance.  Baxxid are unable to chew bubblegum.  There I was, a yinglet a fraction of this creature's size blowing bubbles, and we realized that this was not an inclusive activity.  Requesting baxxid-friendly bubblegum when resources permit.  

Oh, and, of lesser importance, the dope couldn't drink out of a cup even though they asked for a green tea.  I brought them the straw I use for tracheotomies and they were either unwilling (because of the blood) or unable to drink through it.  Either way, I was out of there as quick as could be and did not wish to further examine that horrible mouth.
